lambdas/data-migration2/tests/test-index.js

```diff
@@ -111,22 +105,19 @@ test.beforeEach(async (t) => {
   t.context.testExecution = testExecution;
 });
 
-test.afterEach.always(async (t) => {
+test.after.always(async (t) => {
   await t.context.knex(tableNames.files).del();
```

Comment:
I don't think you need to do these table deletes explicitly - as you're calling destroyLocalTestDb which destroys the postgres database (and therefore all the tables)

Reply:
Do you mean that I don't need to delete them explicitly? If so, that makes sense to me and I will update accordingly.

Reply:
[`37fd579` (#2140)](https://github.com/nasa/cumulus/pull/2140/commits/37fd5790f5751eefd53df71f0c92ec5f4e2f3067)

Reply:
Right, exactly.

Reply:
Sorry, reading back at your comment now I realize I can't read which is why I asked the clarifying question because I thought I had read that I need to do the table deletes explicitly 🚶‍♀️ ☕